xalia/Viewer/UiDomViewer.Designer.cs

```diff
@@ -53,10 +65,82 @@ private void InitializeComponent()
+            this.properties_page.Name = "properties_page";
+            this.properties_page.Padding = new System.Windows.Forms.Padding(3);
+            this.properties_page.Size = new System.Drawing.Size(674, 780);
+            this.properties_page.TabIndex = 0;
+            this.properties_page.Text = "Properties";
+            this.properties_page.UseVisualStyleBackColor = true;
+            // 
+            // properties_view
+            // 
+            this.properties_view.AllowUserToAddRows = false;
+            this.properties_view.AllowUserToDeleteRows = false;
+            this.properties_view.AutoSizeColumnsMode = System.Windows.Forms.DataGridViewAutoSizeColumnsMode.Fill;
+            this.properties_view.AutoSizeRowsMode = System.Windows.Forms.DataGridViewAutoSizeRowsMode.DisplayedCells;
+            this.properties_view.ColumnHeadersHeightSizeMode = System.Windows.Forms.DataGridViewColumnHeadersHeightSizeMode.AutoSize;
+            this.properties_view.ColumnHeadersVisible = false;
+            this.properties_view.Columns.AddRange(new System.Windows.Forms.DataGridViewColumn[] {
+            this.property_column,
+            this.value_column,
+            this.action_column});
+            this.properties_view.Dock = System.Windows.Forms.DockStyle.Fill;
+            this.properties_view.Location = new System.Drawing.Point(3, 3);
+            this.properties_view.Name = "properties_view";
+            this.properties_view.RowHeadersVisible = false;
+            this.properties_view.RowHeadersWidth = 82;
+            this.properties_view.RowTemplate.Height = 33;
+            this.properties_view.SelectionMode = System.Windows.Forms.DataGridViewSelectionMode.CellSelect;
+            this.properties_view.Size = new System.Drawing.Size(668, 774);
+            this.properties_view.TabIndex = 0;
+            this.properties_view.CellClick += new System.Windows.Forms.DataGridViewCellEventHandler(this.properties_view_CellClick);
+            // 
+            // property_column
+            // 
+            this.property_column.AutoSizeMode = System.Windows.Forms.DataGridViewAutoSizeColumnMode.DisplayedCells;
+            this.property_column.HeaderText = "Property";
+            this.property_column.MinimumWidth = 10;
+            this.property_column.Name = "property_column";
+            this.property_column.ReadOnly = true;
+            this.property_column.Width = 10;
+            // 
+            // value_column
+            // 
+            this.value_column.AutoSizeMode = System.Windows.Forms.DataGridViewAutoSizeColumnMode.Fill;
+            this.value_column.HeaderText = "Value";
+            this.value_column.MinimumWidth = 10;
+            this.value_column.Name = "value_column";
+            // 
+            // action_column
+            // 
+            this.action_column.AutoSizeMode = System.Windows.Forms.DataGridViewAutoSizeColumnMode.AllCellsExceptHeader;
+            this.action_column.HeaderText = "action_column";

```

Comment:
Should this be `"Action"` to match the `"property_column"` `HeaderText/Name` string pattern?

Reply:
Well, in theory the text is only visible to accessibility clients, but that's still a good reason to fill it in. But I'm not sure if "Action" is a good description.

Reply:
I went ahead and added that, and I added text to all the buttons (and, the ones that previously didn't do anything now copy the data to the clipboard just so I can label them as such).